operatoringress/v1/types.go

```diff
@@ -115,6 +123,21 @@ const (
 	ARecordType DNSRecordType = "A"
 )
 
+// DNSManagementPolicy is a policy for configuring how the dns controller
+// manages DNSRecords.
+//
+// +kubebuilder:validation:Enum=Managed;Unmanaged
```

Comment:
Because you've specified this here, you don't need to specify it each time you use the enum. If you look at the generated CRD, you have the enum twice because of this